content/docs/components/training/chainer.md

```diff
@@ -1,8 +1,8 @@
 +++
 title = "Chainer Training"
-description = "See v0.7 docs for instrustions on using Chainer for training"
+description = "See v0.6 docs for instrustions on using Chainer for training"
 weight = 4
 toc = true
 +++
 
-[Chainer](https://github.com/kubeflow/chainer-operator) is not supported in kubeflow 1.0
+[Chainer](https://github.com/kubeflow/chainer-operator) is not supported in kubeflow > 0.6
```

Comment:
It's best not to use shorthand or abbreviations like `>` in technical docs. The abbreviations can be confusing and can cause errors in translation.

Change "> 0.6" to "versions greater than 0.6"

Also add a full stop at the end of the sentence.